src/map/script.c

```diff
@@ -15909,6 +15909,36 @@ BUILDIN_FUNC(distance)
 
 // <--- [zBuffer] List of mathematics commands
 
+BUILDIN_FUNC(min)
+{
+	int i, min;
+
+	min = script_getnum(st, 2);
+	for (i = 3; script_hasdata(st, i); i++) {
+		int next = script_getnum(st, i);
+		if (next < min)
+			min = next;
+	}
+	script_pushint(st, min);
+
+	return true;
```

Comment:
Could you adjust this to `return SCRIPT_CMD_SUCCESS;` so it conforms with the rest of the script commands? Thanks!